syne_tune/optimizer/schedulers/searchers/dyhpo/dyhpo_searcher.py

```diff
@@ -92,9 +101,64 @@ def _get_config_modelbased(
         self._debug_log = None  # No more debug logging for rest of ``get_config``
         return {INTERNAL_KEY: "dummy"}
 
+    def _extended_configs_from_paused(
```

Comment:
This is a small refactoring, because the old code was hard to read.

Reply:
Nice change!